.github/workflows/wheels.yml

```diff
@@ -0,0 +1,213 @@
+        with:
+          platforms: all
+
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.9.0
+        # build all wheels here, but the macosx_arm64 job in its own entry
+        # cibuildwheel is currently unable to pass configuration flags to
+        # CIBW_BUILD_FRONTEND (pip/build), which are required to configure
+        # the cross-compile with meson.
+        if: >-
+          ( ! contains(matrix.buildplat[2], 'arm64' ) )
+        env:
+          CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}
+          CIBW_ARCHS: ${{ matrix.buildplat[2] }}
+          CIBW_ENVIRONMENT_PASS_LINUX: RUNNER_OS
+
+      - name: Build macosx_arm64
+        if: ${{ matrix.buildplat[1] == 'macosx*' && matrix.buildplat[2] == 'arm64' }}
+        run: |
+          export PLAT="arm64"
+          export _PYTHON_HOST_PLATFORM="macosx-11.0-arm64"
+          export CROSS_COMPILE=1
+
+          # Need macOS >= 11 for arm compilation.
+          export MACOSX_DEPLOYMENT_TARGET=11.0
+
+          # SDK root needs to be set early, installation of gfortran/openblas
+          # needs to go in the correct location.
+          export SDKROOT=/Applications/Xcode_12.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.1.sdk
+          export ARCHFLAGS=" -arch arm64 "
+
+          # install dependencies for the build machine
+          pip install meson cython pybind11 pythran ninja oldest-supported-numpy numpy build delocate meson-python
+
+          # sets up gfortran compiler/openblas, sets compiler flags.
+          bash tools/wheels/cibw_before_build_macos.sh $(pwd)
+          export PKG_CONFIG_PATH=/opt/arm64-builds/lib/pkgconfig
+          export PKG_CONFIG=/usr/local/bin/pkg-config
+          export CFLAGS=" -arch arm64 $CFLAGS"
+          export CXXFLAGS=" -arch arm64 $CXXFLAGS"
+          export LD_LIBRARY_PATH="/opt/arm64-builds/lib:$FC_LIBDIR:$LD_LIBRARY_PATH"
+          meson setup --cross-file $(pwd)/tools/wheels/cross_arm64.txt build
+
+          # use the pip frontend because the build front end does not end up
+          # obeying the configuration flags it's passed.
+          # For example: `python -m build -Cbuilddir=dir` does not end up using
+          # dir as the meson build directory. This is an issue because
+          # the cross-compile specification is contained in that directory.
+
+          python -m pip wheel -w dist -vvv --config-settings builddir=build .
+          rm dist/numpy*.whl
+
+          # The `.so` are all converted to `-rpath/libgfortran` by
+          # gfortran/meson, with all absolute paths removed.
+          # Enables delocate to find the libopenblas/libgfortran libraries.
+          export DYLD_LIBRARY_PATH=/opt/gfortran-darwin-arm64/lib/gcc/arm64-apple-darwin20.0.0/10.2.1:/opt/arm64-builds/lib
+
+          delocate-listdeps dist/scipy*.whl
+          delocate-wheel --require-archs=arm64 -k -w wheelhouse dist/scipy*.whl
+
+      - uses: kittaakos/upload-artifact-as-is@v0
```

Comment:
This action's repo is archived: https://github.com/kittaakos/upload-artifact-as-is. And it seems rarely used. Is there no better alternative?

Reply:
Now uploading in the form cp38-macosx which would be a zip file containing the arm64 and x86_64 wheels.